rfc/src/rfcs/0001-mir-linker.md

```diff
@@ -108,11 +108,12 @@ where:
  reachability analysis. No goto-program will be generated.
   This will be used to compile dependencies up to the MIR level.
    `kani-compiler` will still generate artifacts with the crate's MIR.
+ - `tests`: Use the functions marked as tests with `#[tests]` as the starting points for the analysis.
  - `legacy`: Keep `kani-compiler` current behavior by using
```

Comment:
Same here.

Reply:
I could update this to say that this is the behavior similar to how `rustc` compile crates into separate objects.

Reply:
That would be useful.

Reply:
Done. Let me know what you think.

Reply:
Looks good. Thanks.